src/multiplayer-servers/getting-started/cloud-provider-setup.md

```diff
@@ -1,7 +1,10 @@
 ## Configuring your Cloud Provider
 
-This document describes the steps required to set up your Cloud provider for use with GameFabric. Support is 
-currently limited to Google Cloud.
+This document describes the steps required to set up your Cloud provider for use with GameFabric. Currently supported
+Cloud providers are Google Cloud, Azure, and AWS.
```

Comment:
```suggestion
Cloud providers are Google Cloud (GCP), Azure, and Amazon Web Services (AWS).
```

Reply:
Not a big fan of mixing abbrevs with full names here.. does azure have a common abbrev tho?

Reply:
Maybe we can also make use of `<abbr>` or `<acronym>` here. Not sure how well that is supported in vitepress.